exec/jira.R

```diff
@@ -0,0 +1,95 @@
+#!/usr/local/bin/Rscript
+
+# Kaiaulu - https://github.com/sailuh/kaiaulu
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+require(yaml,quietly=TRUE)
+require(cli,quietly=TRUE)
+require(docopt,quietly=TRUE)
+require(kaiaulu,quietly=TRUE)
+require(data.table,quietly=TRUE)
+
+
+# Rscript jira.R refresh issues ../../kaiaulu/tools.yml ../../kaiaulu/conf/geronimo.yml ../../rawdata/issue_tracker/geronimo/issue_comments/
+
+doc <- "
+USAGE:
+  jira.R refresh help
+  jira.R refresh issues <tools.yml> <project_conf.yml> <save_file_name_path>
+  jira.R download modmbox help
```

Comment:
Note to self: Should remove this from here now we have a mailinglist exec that handles this with refresher.